### PR TITLE
 How to add custom stamp to the menu item from the sample;  How to add an annotation in code behind at run time in WPF PDF Viewer

### PR DESCRIPTION
**Pull request by RajaVigneshB**
**Added the below samples:**

- Sample for adding custom stamp to the menu item of the toolbar.
- Sample for adding annotation at the runtime.

### CODE REVIEW

File: Annotations/Add Annotation/WpfPDFViewer/App.config [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
+    </startup>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Syncfusion.Pdf.Base" publicKeyToken="3d67ed1f87d44c89" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-16.4460.0.1" newVersion="16.4460.0.1" />
```
Comment (Deepak1799) [inline Annotations/Add Annotation/WpfPDFViewer/App.config:10]: Please refer assemblies from Nuget.packages alone

File: Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.Win32;
+using Syncfusion.Pdf;
```
Comment (Deepak1799) [inline Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs:2]: Please remove unwanted namespaces

File: Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            //Gets the loadedDocument from PDF Viewer
+            PdfLoadedDocument loadedDocument = pdfViewer.LoadedDocument;
+
+            //Specifies the ink points
+            List<float> linePoints = new List<float> { 40, 300, 60, 100, 40, 50, 40, 300 };
+
+            //Creates a new ink annotation
+
+            RectangleF rectangle = new RectangleF(0, 0, 300, 400);
```
Comment (Deepak1799) [inline Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs:50]: Explain what this rectangle represents.

File: Toolbar/Add Stamps to Menu/Sample_Project/packages.config [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Deepak1799) [inline Toolbar/Add Stamps to Menu/Sample_Project/packages.config:1]: image

Name the project name meaningfully and also keep the sln and .csprojec in the same folder

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Annotations/Add Annotation/WpfPDFViewer/App.config (hunk 1 above)
Comment (RajaVigneshB) [reply Annotations/Add Annotation/WpfPDFViewer/App.config:10]: Okay sir. Now added properly.

File: Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs (hunk 2 above)
Comment (RajaVigneshB) [reply Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs:2]: Removed sir

File: Toolbar/Add Stamps to Menu/Sample_Project/packages.config (hunk 4 above)
Comment (RajaVigneshB) [reply Toolbar/Add Stamps to Menu/Sample_Project/packages.config:1]: Changed the name and moved both in the same folder sir.

File: Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs (hunk 3 above)
Comment (RajaVigneshB) [reply Annotations/Add Annotation/WpfPDFViewer/MainWindow.xaml.cs]: Mentioned it sir.

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

**Comment by Deepak1799** [comment]
Samples are added

**Pull request merged**